experimental/rhscl-postgresql-apb/README.md

```diff
@@ -0,0 +1,23 @@
+rhscl-postgresql-apb
+======================
+
+An apb for deploying [PostgreSQL](https://www.postgresql.org).
+
+## What it does
+* Deploys 1 postgresql container.
```

Comment:
Can you be a little more descriptive as to why this is different than rhscl-postgresql-apb? We want to make sure the user understands the feature demonstrated by this playbook.